beta/src/content/docs/en/api/effector/Event.md

```diff
@@ -9,131 +9,272 @@ redirectFrom:
   - /docs/api/effector/event
 ---
 
-_Event_ is an intention to change the state. Let's imagine a life situation, you enter a shop and, according to etiquette, you have to say "hello" – **intention**, then you say "hello" – **event**.
+The **Event** in effector represents a user action, a step in the application process, a command to execute, or an intention to make modifications, among other things. This unit is designed to be a carrier of information/intention/state within the application, not the holder of a state.
 
-Event calls always return its payload:
+There are two type of events provided by effector: [`Event`](#event) and [`ReadonlyEvent`](#readonlyEvent).
 
-```js
+# Event instance {#event}
+
+In most situations, it is recommended to create events directly within the module, rather than placing them within conditional statements or classes, in order to maintain simplicity and readability. An exception to this recommendation is the use of factory functions; however, these should also be invoked at the root level of the module.
+
+:::info
+Event instances persist throughout the entire runtime of the application and inherently represent a portion of the business logic.
+
+Attempting to delete instances and clear memory for the purpose of saving resources is not advised, as it may adversely impact the functionality and performance of the application.
```

Comment:
Probably this part is not really needed here - it just adds to much of additional complexity to explanation. I think, it should be explained somewhere else, as a separate article